src/main/java/stsjorbsmod/powers/StrengthNextTurnPower.java

```diff
@@ -0,0 +1,53 @@
+package stsjorbsmod.powers;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.util.TextureLoader;
+
+import static stsjorbsmod.JorbsMod.makePowerPath;
+
+public class StrengthNextTurnPower extends CustomJorbsModPower {
+    public static final String POWER_ID = JorbsMod.makeID(StrengthNextTurnPower.class.getSimpleName());
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    private static final Texture tex84 = TextureLoader.getTexture(makePowerPath("strength_next_turn_power84.png"));
+    private static final Texture tex32 = TextureLoader.getTexture(makePowerPath("strength_next_turn_power32.png"));
+
+    private int damageAbsorbedThisTurn = 0;
+
+    public StrengthNextTurnPower(AbstractCreature owner, int amount) {
+        super(StaticPowerInfo.Load(StrengthNextTurnPower.class));
```

Comment:
`StaticPowerInfo` is intended to be loaded into a static field, rather than each time it's constructed; loading the info is a little expensive, so it avoids some frame jitter. I still need to update the other existing powers to use the intended strategy, but `CoilPower` is a good example of how it's intended to be used